src/Resources/views/base_layout.html.twig

```diff
@@ -37,7 +30,6 @@ file that was distributed with this source code.
     {% endblock %}
 
         {% block sonata_page_top_bar %}
-            {% block page_top_bar %} {# Deprecated block #}
                 {% if sonata_page.isEditor or ( app.user and is_granted('ROLE_PREVIOUS_ADMIN') ) %}
```

Comment:
can you change indentation too?

Reply:
Hey @jordisala1991 I didn't change the indentation just for be better to you review, but if every thing is fine I can fix the indentation :)